generators/app/templates/_README.md

```diff
@@ -54,10 +54,25 @@ player.<%= pluginFunctionName %>();
 When using with RequireJS (or another AMD library), get the script in whatever way you prefer and `require` the plugin as you normally would:
 
 ```js
-require(['video.js', '<%= packageName %>'], function(videojs) {
-  var player = videojs('my-video');
+// configure the paths to the packages with requirejs
+requirejs.config({
+  paths: {
+    "videojs": "videojs-url",
+    "<%= pluginFunctionName %>": "plugin-url",
```

Comment:
It seems wrong that we'd have to configure paths at all. I feel like a UMD build should "just work".